### PR TITLE
[ci] Silence migration tests

### LINKED ISSUE #4859: Remove warnings from rspec testsuite
Issue/Feature description
====================
When running the rspec testsuite, it prints several warnings to stdout which breaks the outut (and is annoying as you might miss some important output). Examples:

```
WARNING: rspec-mocks was unable to restore the original `full_path` method on #<Backend::File:0x000000000d78af98 @name="fake_filename", @file=nil, @response={:type=>"text/xml", :status=>"200", :size=>211}, @last_read_query={}> because it has been frozen.  If you reuse this object, `full_path` will continue to respond with its stub implementation. Warning generated from spec at `./spec/models/backend/file_spec.rb:225`
```
(from spec/models/backend/file_spec.rb)
or

```
-- change_column_default(:repositories, :remote_project_name, "")
   -> 0.0040s
```
(from spec/db/data/remove_duplicate_repositories_spec.rb)

There is also one deprecation warning from shoulda which we can not fix as there is no newer version yet (it is already fixed in master). Also there is a syntax error in  /usr/lib/perl5/vendor_perl/5.18.2/x86_64-linux-thread-multi/XML/Parser.pm line 187. I would ignore these two warnings for now.

Expected result
============
- spec/models/backend/file_spec.rb specs are fixed so that they do not throw a warning
- Output from spec/db/data/remove_duplicate_repositories_spec.rb does not appear in stdout anymore (e.g. redirect it to /dev/null)

### PR DESCRIPTION
**Pull request by coolo**
Fixes #4859

### CODE REVIEW

File: src/api/spec/db/data/remove_duplicate_repositories_spec.rb [hunk 1]
```diff
@@ -9,6 +9,10 @@
   let(:schema_migration) { ChangeRepositoriesRemoteProjectNameToNotNull.new }
   let(:data_migration) { RemoveDuplicateRepositories.new }
 
+  before do
+    ActiveRecord::Migration.verbose = false
```
Comment (bgeuken) [inline src/api/spec/db/data/remove_duplicate_repositories_spec.rb]: Doesn't this change the verbosity of all following tests (test files)?

Comment (bgeuken) [reply src/api/spec/db/data/remove_duplicate_repositories_spec.rb]: Then I would move this to the rspec configuration, somewhere in [support](https://github.com/openSUSE/open-build-service/tree/master/src/api/spec/support) probably, or the rails test environment.
Right now, any data migration test that runs before this one would still have the verbosity turned on.

**Comment by coolo** [comment]
@bgeuken like so?

**Comment by coolo** [comment]
Assume so

**Pull request merged**